components/feature/prompts/README.md

```diff
@@ -15,14 +15,46 @@ implementation "org.mozilla.components:feature-prompts:{latest-version}"
 ### PromptFeature
 
   ```kotlin
-  val promptFeature = PromptFeature(sessionManager, fragmentManager)
+
+  val onNeedToRequestPermissions : ( Session, Array<String>, Int) -> Unit = {
+    session, permissions, requestCode ->
+    /* You are in charge of triggering the request for the permissions needed,
+     * this way you can control, when you request the permissions,
+     * in case that you want to show an informative dialog, '
+     * to clarify the use of these permissions.
+     */
+    this.requestPermissions(permissions, requestCode)
+  }
+
+  val promptFeature = PromptFeature(fragment = this,
+    sessionManager = sessionManager,
+    fragmentManager= fragmentManager,
+    onNeedToRequestPermissions = onNeedToRequestPermissions
+  )
 
   //It will start listing for new prompt requests for web content.
   promptFeature.start()
 
   //It will stop listing for future prompt requests for web content.
   promptFeature.stop()
 
+  /* There some requests that are not handled with dialogs, instead they are delegated to other apps
+   * to perform the request, an example is a file picker request, that delegates to the OS file picker.
+   * For this reason, you have to inform the result of this petitions to the promptFeature by overriding,
```

Comment:
nit ...`you have to forward the results of these requests to the prompt feature....`